ambry-commons/src/main/java/com/github/ambry/commons/CommonUtils.java

```diff
@@ -38,9 +35,7 @@ public static HelixPropertyStore<ZNRecord> createHelixPropertyStore(String zkSer
     if (zkServers == null || zkServers.isEmpty() || propertyStoreConfig == null) {
       throw new IllegalArgumentException("Invalid arguments, cannot create HelixPropertyStore");
     }
-    ZkClient zkClient = new ZkClient(zkServers, propertyStoreConfig.zkClientSessionTimeoutMs,
-        propertyStoreConfig.zkClientConnectionTimeoutMs, new ZNRecordSerializer());
-    return new ZkHelixPropertyStore<>(new ZkBaseDataAccessor<>(zkClient), propertyStoreConfig.rootPath,
+    return new ZkHelixPropertyStore<ZNRecord>(zkServers, new ZNRecordSerializer(), propertyStoreConfig.rootPath,
```

Comment:
Overall lgtm, but can this line be replaced with this since the code in the two methods is same:

>     return createHelixPropertyStore(zkServers, propertyStoreConfig.rootPath, subscribedPaths);